modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/main/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/PowFunction.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.dynamic.data.mapping.form.evaluator.internal.function;
+
+import com.liferay.dynamic.data.mapping.expression.DDMExpressionFunction;
+
+import java.math.BigDecimal;
+
+/**
+ * @author Selton Guedes
+ */
+public class PowFunction
+	implements DDMExpressionFunction.Function2<Object, Object, BigDecimal> {
```

Comment:
Could we use like this: <BigDecimal, BigDecimal, BigDecimal> ?

Reply:
We can't use the BigDecimal because the parameter can be an Integer and it can't be directly converted to a BigDecimal